lua/neo-tree/setup/netrw.lua

```diff
@@ -5,14 +5,16 @@ local command = require("neo-tree.command")
 local M = {}
 
 local get_position = function(source_name)
-  local pos = utils.get_value(M, "config." .. source_name .. ".window.position", "left")
+  local nt = require("neo-tree")
+  local default_pos = utils.get_value(nt.config, "window.position", "left", true)
+  local pos = utils.get_value(nt.config, source_name .. ".window.position", default_pos, true)
```

Comment:
This part should not be necessary as the global default should have already been merged down to the source config by the initial call to `setup()`. If that does not currently work, it should be fixed here:
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/039aff445280dd4ffce3c4b97e25d7962c0bcfd4/lua/neo-tree/setup/init.lua#L572-L577

Reply:
It seems that `M.config` in `netrw.lua` is never set to the global config, thus why getting position from `nt.config` worked. My latest commit changes that in the `setup()` function which also fixes netrw hijack, though I'm really not sure if this is the right solution.

https://github.com/nvim-neo-tree/neo-tree.nvim/pull/874/files#diff-3032d6dddfe6b408e380eae368f03366da4adcc797cc2b8e05a27420573efa72

Reply:
Actually, the netrw module is not a source at all and should not have any config. the config belongs to the sources, such as `filesystem`, `buffers`, `git_status`, etc. That is what is being pulled by:
```
local pos = utils.get_value(nt.config, source_name .. ".window.position", "left", true)
```
and that section of code should not have been touched at all. The global config is already being applied to the source config in the lines of code that I quoted originally. **If** that code has a bug, it should be fixed at this location: 
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/039aff445280dd4ffce3c4b97e25d7962c0bcfd4/lua/neo-tree/setup/init.lua#L572-L577

I would double check that though, because I don't think there is any bug to be fixed there.